card/mod/02_basic_types/set/all/base.rb

```diff
@@ -89,34 +86,38 @@ def show view, args
     end.inspect
   end
 
-
-
-  #none of the below belongs here!!
-
+  # none of the below belongs here!!
 
   view :template_rule, tags: :unknown_ok do |args|
```

Comment:
I don't think this method is well covered in our tests.  I probably have to remedy that before I can merge this.